snowpack/src/commands/install.ts

```diff
@@ -359,6 +359,10 @@ ${colors.dim(
       polyfillNode && rollupPluginNodePolyfills(),
       ...userDefinedRollup.plugins, // load user-defined plugins last
       rollupPluginCatchUnresolved(),
+      {
+        name: 'strip-source-mapping',
+        transform: (code) => code.replace(/\/\/+#\s*sourceMappingURL=.+$/gm, ''),
+      },
```

Comment:
It's a simple enough plugin to do it inline, but moving it into its own file would be more consistent with the other plugins. I'd also suggest to add a comment that references https://github.com/pikapkg/snowpack/issues/932